### PR TITLE
Keyboard Navigation Fixes

### PR DESCRIPTION
**Pull request by nisarhassan12**
Made section "Three Ways to start coding" accessible for Keyboard Users by adding Keyboard support
image

### CODE REVIEW

**Comment by svenefftinge** [comment]
I see blue focus boxes on the elements, but switching using cursor keys didn't work for me.
Screenshot 2019-06-22 at 20 17 06

**Comment by nisarhassan12** [comment]
@svenefftinge I haven't added the support for cursor keys. I just made it to work with Tab(to move forwards ) and Shift + Tab(to move backwards). Is it necessary to support cursor keys as well?

**Comment by svenefftinge** [comment]
>  Is it necessary to support cursor keys as well?
No it's not, I just assumed that was what you had done.

File: src/pages/index.tsx [hunk 1]
```diff
@@ -206,6 +206,12 @@ const SectionBuiltOn = styled.section`
     }
 `
 
+const UnstyledButton = styled.button`
+    border: none;
+    padding: 0;
+    margin: 0;
```
Comment (jankeromnes) [inline src/pages/index.tsx]: The blue outline @svenefftinge showed is actually visible in Chrome when you click on any of these items.

I don't think it looks good, so we should also "unstyle" the `outline` property:

```suggestion
    margin: 0;
    outline: 0;
```

**Pull request merged**